docs/kata-containers.md

```diff
@@ -0,0 +1,44 @@
+# Kata containers support in CRI-RM
+
+[Kata Containers](https://katacontainers.io/) is an open source container runtime, 
+building lightweight virtual machines that seamlessly plug into the containers ecosystem.
+
+In order to enable Kata container in a Kubernetes-CRI-RM stack, both Kubernetes 
+and the Container Runtime Engine need to be aware of the new runtime environment:
+
+  * The Container Runtime Engine can only be CRI-O or containerd, and will need to
+   have the runtimes enabled in their configuration files.
+  * Kubernetes will also have to be made aware via a "RuntimeClass"  resource
```

Comment:
aware ..._of smth_ via

Also, link to kubernetes documentation (about enabling/configuring runtime classes) would be good